symbolicationRequest.py

```diff
@@ -19,7 +19,10 @@ def __init__(self, libName, breakpadId):
     self.libName = libName
     self.breakpadId = breakpadId
 
-def getModuleV2(libName, pdbAge, pdbSig, pdbName):
+def getModuleV2(libName, pdbAge, pdbSig, pdbName, remoteIp):
```

Comment:
I'm not concerned about detailed error logging from getModuleV2, just log the IP in the SymbolicationRequest code that calls getModuleV2